docs/installingNative.md

```diff
@@ -71,22 +71,11 @@ Now you can install the other dependencies.
 
 === "macOS"
 
-    Brew can install the rest of the dependencies.
+    Brew can install the rest of the dependencies. Enabling `corepack` makes `yarn` available.
 
     ```sh
-    brew install node@20 python@3.10
-    ```
-
-    You can install yarn through npm:
-
-    ```sh
-    npm install -g yarn
-    ```
-
-    d2 can be installed through the install script:
-
-    ```sh
-    curl -fsSL https://d2lang.com/install.sh | sh -s --
+    brew install node npm
+    corepack enable
```

Comment:
The [`yarn` docs](https://yarnpkg.com/getting-started/install) prefer installing via `corepack`.

Reply:
Node will be removing `corepack` in the future: https://github.com/nodejs/package-maintenance/pull/606. I'm fine to use it until Yarn changes their recommendation.